unlock-app/src/components/content/event/EventDetails.tsx

```diff
@@ -571,7 +573,10 @@ export const EventDetails = ({ lockAddress, network }: EventDetailsProps) => {
         <Checkout
           injectedProvider={injectedProvider as any}
           paywallConfig={paywallConfig}
-          handleClose={() => setCheckoutOpen(false)}
+          handleClose={() => {
```

Comment:
I am confused about that change? Can you explain what this does and why you are adding it here?

Reply:
the idea is to refetch the data once the modal is closed, and when a purchase happens we can see this message `You already have a ticket! You can view it in` without any page reload.